docs/Third-party.md

```diff
@@ -1,281 +1,268 @@
+    **Please** do not copy/paste the provided commands without adapting them to your system and without testing them.
+    Most paths provided here require to be adapted to your system and depend on how you installed the software.
+    
+    A good approach is to verify that the different paths you want to assign to a variable exist first : 
 
-_*You only need to install ITK if you plan to perform partial volume correction using PETPVC._
+    :material-arrow-right: If they do, then try running the `export` and `source` **(1.)** commands in your terminal. Verify that the software or the pipeline you want runs as expected.
+    
+    :material-arrow-right: If this works, then consider modifying your shell configuration file **(2.)** to have these variables automatically defined on every session.
 
-Depending on the architecture and OS of your system, setup of third party libraries can change.
-Please refer to each tool’s website for installation instructions:
 
-### Environment variables setup
+## Converters
 
-When installing some of the third party software, environment variables might be needed in order to configure some installations.
+Some converters require **dcm2niix** to transform DICOM files into NIfTI :
 
-If you simply run the `export` and `source` commands in your terminal, these environment variables will be defined only for the duration of your session, and will be lost if you re-launch your terminal.
+<div class="grid">
+  <a href="Converters/ADNI2BIDS.md" class="card">adni-to-bids</a>
+  <a href="Converters/AIBL2BIDS.md" class="card">aibl-to-bids</a>
+  <a href="Converters/NIFD2BIDS.md" class="card">nifd-to-bids</a>
+  <a href="Converters/UKBtoBIDS.md" class="card">ukb-to-bids</a>
+</div>
 
-In order to define these variables permanently, you need to manually edit the configuration file associated to your shell.
+!!! warning "Version required"
+    Clinica requires dcm2niix version `1.0.20190902` or later.
 
-If you are using `bash`, it can be `~/.bashrc` or `~/.bash_profile`, if you are using `zsh`, it should be `~/.zshrc`.
+Please check the installation instructions for all platforms on [dcm2niix Git repository](https://github.com/rordenlab/dcm2niix#install).
 
-!!! note "test before"
-    Please do not copy/paste the provided commands without adapting them to your system and without testing them.
-    Most paths provided here require to be adapted to your system, and also depends on how you installed the software.
-    A good approach is to verify that the different paths you want to assign to a variable exist first.
-    If they do, then try running the `export` and `source` commands in your terminal, and verify that the software run as expected.
-    You can also verify that the pipeline you want to use is also running as expected.
-    If this works, then consider modifying your shell configuration file to have these variables automatically defined on every session.
 
 
-### ANTs
+## Pipeline-specific interfaces
```

Comment:
Maybe we can simplify this to "Pipelines", which would make sense since we have a "Converters" section juste above. WDYT ?